MiaServiceDotNetLibrary.Tests/Environment/ConfigValidatorTest.cs

```diff
@@ -35,7 +34,6 @@ public void TestInvalidConfigOnePropNull()
         {
             var config = new MiaEnvConfiguration
             {
-                USERID_HEADER_KEY = "userid",
```

Comment:
Why have you removed this variable?

Reply:
Because the test is testing what happens if one variable is not received, and previously the one missing was CRUD_PATH, so I had to remove one and I chose USERID_HEADER_KEY

Reply:
ok